sdk/schemaregistry/client/pom.xml

```diff
@@ -0,0 +1,49 @@
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>schemaregistry-parent</artifactId>
+    <version>0.5-preview</version>
+  </parent>
+
+  <artifactId>schemaregistry-client</artifactId>
+  <packaging>jar</packaging>
+  <version>0.5-preview</version>
+  <name>schemaregistry-client</name>
+
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+  
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
```

Comment:
To add to what @srnagar said. The [analyze step is failing](https://dev.azure.com/azure-sdk/public/_build/results?buildId=397954&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=bdeefc16-b669-5ebd-ad94-a2c19ade53b0&l=75) because of the following client libraries need to be defined in [version_client.txt](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_client.txt):
azure-schemaregistry-client
azure-schemaregistry-serde-avro
azure-schemaregistry-serde-common
The pom files need the appropriate tags and for the above libraries, within the sdk/schemaregistry they'll all be the current version. You'll also need to add the libraries to the [eng/jacoco-test-coverage](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/jacoco-test-coverage/pom.xml
)
The following [external_dependencies](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/external_dependencies.txt) need some work.
You'd added org.apache.avro:avro and tagged the bannedDependencies correctly in the pom files however the version tags for the dependencies, themselves are missing. 
Similarly, org.easymock:easymock needs to get added to external_dependencies and correctly tagged within the pom files. 
Last but not least, junit:junit dependencies need to be tagged but you're using version 4.12 and it would be great if, instead of declaring a one off, you could use the common 4.13-beta-3 version that's already defined.

Fixing the above issues should allow the analyze step to pass as well as the From Source runs. Note: You can run the [pom_file_version_scanner.ps1](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/pom_file_version_scanner.ps1#L30) locally from a pwsh or powershell window. The line I linked to has the exact command and it takes like 5 seconds to run.

Reply:
Thanks for the comment Jim.  I'm still in the process of fixing all of Srikanta's comments but this will be helpful.

I'll also be replacing any easymock use with mockito as Srikanta requested.  I've also decided to just convert all junit 4 classes to use junit-jupiter-api.